src/main/java/uk/ac/ebi/eva/contigalias/controller/ContigAliasController.java

```diff
@@ -47,31 +47,49 @@ public ContigAliasController(AssemblyService assemblyService, ChromosomeService
         this.chromosomeService = chromosomeService;
     }
 
-    @ApiOperation(value = "Get an assembly using its Genbank or Refseq accession.")
+    @ApiOperation(value = "Get an assembly using its Genbank or Refseq accession. ",
+            notes = "Given an assembly's accession this endpoint will return an assembly that matches that accession." +
+                    "The accession can be either a genbank or refseq accession and the endpoint will automatically " +
+                    "fetch a result from the database for any assembly having the accession as it's " +
+                    "genbank or refseq accession. " +
+                    "This endpoint will either return an empty list when no result is found or at most " +
```

Comment:
Is that what will happen? If not found, I don't think we should return an empty list, just a 404.

Reply:
The controller indeed returns a 404 instead of an empty list. I had mistakenly written the documentation based on the implementation in the Service and not the Controller. Have fixed it now.